packages/realm/src/Realm.ts

```diff
@@ -18,6 +18,7 @@
 
 import {
   AggregatePipelineStage,
+  AnyRealmObject,
```

Comment:
Perhaps not a part of this PR, but should this also be exported to users via named export? Looks like it's missing from `packages/realm/src/index.ts`. @kraenhansen

Reply:
Ah okay, even if the public APIs use that type in the signatures?